src/xdocs/config_imports.xml

```diff
@@ -1436,11 +1436,21 @@ import java.util.stream.IntStream;
             <td>sortStaticImportsAlphabetically</td>
             <td>whether <b>static imports</b> grouped by <b>top</b> or <b>bottom</b> option
                 are sorted alphabetically or not.
-                Attention: It is applied to all static imports as one group.</td>
+                Attention: It is applied to all static imports as one group
+                when the value of separatedStaticGroups is false.</td>
```

Comment:
we try to make properties for statics to mirror what we have for simple imports.
Sorting should be done in group. So static imports should follow the same groups as defined for simple groups while there is no `groupsStatic` property or be treated as one single group unrelated to groups of simple imports.

There should be no dependency between separatedStaticGroups and sortStaticImportsAlphabetically.